utils/spacewalk-common-channels

```diff
@@ -289,16 +305,10 @@ Create everything as well as unlimited activation key for every channel:
                                 base_info['label'], e.faultString))
         else:
             # check whether channel exists
-            try:
-                base_info = client.channel.software.getDetails(key,
```

Comment:
It seems I still need base_info, as it is housed below. And `e` does not exist. Fixing it.